charts/dragonfly/templates/dfdaemon/dfdaemon-daemonset.yaml

```diff
@@ -300,7 +300,7 @@ spec:
         - /bin/sh
         - -cx
         - |-
-          etcContainerd=/host/etc/containerd
+          etcContainerd={{ .Values.containerRuntime.containerd.configPathDir }}
```

Comment:
`etcContainerd={{ .Values.containerRuntime.containerd.configPathDir }}` ->
`etcContainerd=/host{{ .Values.containerRuntime.containerd.configPathDir }}`

Reply:
Maybe add `dfdaemon.extraVolumeMounts` to https://github.com/dragonflyoss/helm-charts/blob/01207e893cac17389206898ed9c98108b6eabca0/charts/dragonfly/templates/dfdaemon/dfdaemon-daemonset.yaml#L405-L407 ?

```

        {{- if .Values.dfdaemon.extraVolumeMounts }}
        {{- toYaml .Values.dfdaemon.extraVolumeMounts | nindent 8 }}
        {{- end }}
```

Reply:
I think just replace `/etc` to `{{ .Values.containerRuntime.containerd.configPathDir }}` in volumes  and update `/host/etc ` to `/host{{ .Values.containerRuntime.containerd.configPathDir }}` in volumeMounts. BTW, change `name: etc` to `containerd` or other name.